File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] a) {
       /* int res = nums.length;
        
        for (int i = 0; i < nums.length; i++) {
            res += i - nums[i];
        }
        
        return res;*/
        int n=a.length;
        Arrays.sort(a);
        for(int i=0;i<n;i++){
           if(a[i]!= i){
            return i;
           }
        }   
        return n;    
    }
}

File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* a, int n) {
    for(int i=0;i<n+1;i++){
        int flag =1;
        for(int j=0;j<n;j++){
            if(a[j] == i){
                flag = 0;
                break;
            }
        }
        if(flag==1) return i;
    }
    return 0;
    }
